### DISCUSSION TITLE
百度小程序模板的 循环语法只支持 trackby，为什么 3.x 使用的依然是 s-key 呢？

### DISCUSSION THREAD

**Discussion by souldar** [NervJS/taro#12844] [Q & A]
百度小程序模板的 循环语法只支持 trackby，为什么 3.x 使用的依然是 s-key
https://smartprogram.baidu.com/forum/topic/show/116712
![image](https://user-images.githubusercontent.com/16811530/202441297-6730650b-dd81-4351-bf96-036508c17dc4.png)

百度文档中也没有更新 s-key 相关的语法

**Comment by Chen-jj** [comment]
的确，感谢指正，尽快调整下

**Comment by Chen-jj** [comment]
@souldar 可以麻烦新建一个 issue 方便追踪么～

**Reply by souldar** [reply]
https://github.com/NervJS/taro/issues/12864
